flutter/lib/src/integrations/native_app_start_integration.dart

```diff
@@ -34,14 +34,18 @@ class NativeAppStartIntegration extends Integration<SentryFlutterOptions> {
     _frameCallbackHandler.addPostFrameCallback((timeStamp) async {
       try {
         if (!options.autoAppStart && _appStartEnd == null) {
-          await _appStartEndCompleter.future
-              .timeout(const Duration(seconds: 10));
+          await _appStartEndCompleter.future.timeout(
+            const Duration(seconds: 10),
+          );
+        }
+        final appStartEnd = _appStartEnd;
+        if (appStartEnd != null) {
+          await _nativeAppStartHandler.call(
+            hub,
+            options,
+            appStartEnd: appStartEnd,
+          );
```

Comment:
this doesn't take into account when `autoAppStart == true` that the appStart end is `options.clock()` because this happens right after the frame has rendered

Reply:
🤦